src/late_multicellular_stage/MulticellularConvolutionDispayer.cs

```diff
@@ -75,17 +76,30 @@ public void DisplayFromLayout(IReadOnlyCollection<MulticellularMetaball> layout)
         Mesh = meshGen.DualContour();
         Mesh.SurfaceSetMaterial(0, material);
 
+        Task uvUnwrap = new Task(() => UVUnwrapAndTexturize((Mesh as ArrayMesh)!));
```

Comment:
If the cast must succeed, shouldn't this use the cast syntax that ensures it succeeds or throws (`(ArrayMesh)Mesh`)?